cxbox-core/src/main/java/org/cxbox/core/service/ResponseService.java

```diff
@@ -193,4 +198,22 @@ public interface ResponseService<T extends DataResponseDTO, E extends BaseEntity
 	 */
 	boolean isDeferredCreationSupported(BusinessComponent bc);
 
+	default CrudmaActionType getActionType() {
```

Comment:
no. just introduce new methods. then autowire PlatformRequest bean and call its methods. do not copy paste its code here